pantab/tests/test_roundtrip.py

```diff
@@ -72,5 +72,17 @@ def test_roundtrip_with_external_hyper_process(df, tmp_hyper):
         pantab.frame_to_hyper(df, tmp_hyper, table="test", hyper_process=hyper)
         pantab.frame_from_hyper(tmp_hyper, table="test", hyper_process=hyper)
         pantab.frames_from_hyper(tmp_hyper, hyper_process=hyper)
+        pantab.frame_from_hyper_query(
+            tmp_hyper, "SELECT * FROM test", hyper_process=hyper
+        )
 
     assert not default_log_path.exists()
+
+
+def test_roundtrip_with_external_hyper_connection(df, tmp_hyper):
+    with HyperProcess(Telemetry.DO_NOT_SEND_USAGE_DATA_TO_TABLEAU) as hyper:
+        with Connection(hyper.endpoint, tmp_hyper, CreateMode.CREATE) as connection:
+            pantab.frame_to_hyper(df, connection, table="test")
```

Comment:
Can you make this test a little bit stronger by just placing `tm.assert_frame_equal` calls after each method call? Will just really ensure that we aren't say reading or writing empty records

Reply:
done